crates/sncast/src/state/state_file.rs

```diff
@@ -396,7 +396,8 @@ mod tests {
     }
 
     #[test]
-    #[should_panic(expected = "No such file or directory (os error 2)")]
+    #[should_panic(expected = "Failed to load state file")]
+    // TODO handle system specific panics
```

Comment:
Is it going to be handled in this PR?

Reply:
Please link issue number if it won't be implemented in this PR 🙏

Reply:
No longer needed, we are catching our error now. Removing todo.